src/common/Channel.hpp

```diff
@@ -104,6 +104,7 @@ class Channel : public std::enable_shared_from_this<Channel>
     virtual bool shouldIgnoreHighlights() const;
     virtual bool canReconnect() const;
     virtual void reconnect();
+    virtual QString &logFolderName();
```

Comment:
```suggestion
    virtual const QString &logFolderName();
```

Looks like it's only used as a getter.